### PR TITLE
Update Bigtable for new style rules

### LINKED ISSUE #4008: Update Bigtable .rubocop.yml
The following configuration from [ruby-style](https://github.com/googleapis/ruby-style/blob/master/google-style.yml#L42-L47) should be manually added to `.rubocop.yml`, and source code updated as needed to comply with this style:

```
Style/MethodCallWithArgsParentheses:
  Enabled: true
  EnforcedStyle: omit_parentheses
  AllowParenthesesInMultilineCall: true
  AllowParenthesesInCamelCaseMethod: true
  AllowParenthesesInChaining: true
```

### PR DESCRIPTION
**Pull request by blowmage**
Update the code for the Ruby 2.4+ style guide.

[closes #4008]

### CODE REVIEW

**Comment by quartzmo** [comment]
@blowmage Can you please update for the 4 Bigtable PRs I just merged into master? Thanks!!

**Comment by blowmage** [comment]
Updated.

File: google-cloud-bigtable/lib/google/cloud/bigtable/table/column_family_map.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
```
Comment (dazuma) [inline google-cloud-bigtable/lib/google/cloud/bigtable/table/column_family_map.rb]: This file should not be here any longer.

Comment (quartzmo) [reply google-cloud-bigtable/lib/google/cloud/bigtable/table/column_family_map.rb]: Yes, somehow this got added back in.

**Pull request merged**